doc/python/getting-started.md

```diff
@@ -138,20 +136,19 @@ See [_Displaying Figures in Python_](/python/renderers/) for more information on
 See the [Troubleshooting guide](/python/troubleshooting/) if you run into any problems with JupyterLab, particularly if you are using multiple Python environments inside Jupyter.
 
 
-
 #### Jupyter Notebook Support
 
 For use in the classic [Jupyter Notebook](https://jupyter.org/), install the `notebook` and `ipywidgets`
 packages using `pip`:
 
 ```
-$ pip install "notebook>=5.3" "ipywidgets>=7.5"
+pip install "notebook>=7.0" "anywidget>=0.9.13"
```

Comment:
Should we guard against notebook v8 and higher to avoid potential breakages in feature?

Reply:
I don't think so. We can always update this if there is an issue in a future version.

Reply:
Debatable : )